hands_on/view_graph.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from exercise_utils.file import append_to_file
+from exercise_utils.git import add, commit, init
+
+__requires_git__ = True
+__requires_github__ = False
+
+
+def download(verbose: bool):
+    os.makedirs("things")
+    os.chdir("things")
+
+    init(verbose)
+
+    append_to_file(
+        "fruits.txt",
+        "apples\nbananas\ncherries\ndragon fruits\n"
+    )
```

Comment:
Use create_or_update_file instead, and use a multi-line string: https://github.com/git-mastery/exercises/blob/main/hands_on/first_commit.py#L14-L22